src/Backoffice/City/App/Requests/BaseCityRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\City\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\City\Domain\DataTransferObject\CityDto;
+use Lightit\Backoffice\City\Domain\Models\City;
+
+abstract class BaseCityRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public function rules(): array
+    {
+        return [
+            self::NAME => ['required', 'string', 'max:255', 'unique:' . City::class . ',name'],
```

Comment:
I'm not sure this works, the `City::class` returns `City`, right? you should put `unique:cities,name`